### PR TITLE
chore: keeps repositories in the tmp dir

### PR DESCRIPTION
**Pull request by bartoszmajsak**
## Description
Which is wiped out when script exits.

## Merge criteria:

- [ ] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

File: get_all_manifests.sh [hunk 1]
```diff
@@ -43,10 +43,11 @@ if [ "$#" -ge 1 ]; then
     done
 fi
 
-# R.I.P, odh-manifests
+TMP_DIR=$(mktemp -d -t "odh-manifests.XXXXXXXXXX")
+trap '{ rm -rf -- "$TMP_DIR"; }' EXIT
 
 for key in "${!COMPONENT_MANIFESTS[@]}"; do
-    echo "Cloning repo ${key}: ${COMPONENT_MANIFESTS[$key]}"
+    echo -e "\033[32mCloning repo \033[33m${key}\033[32m:\033[0m ${COMPONENT_MANIFESTS[$key]}"
```
Comment (zdtsw) [inline get_all_manifests.sh:50]: ?

Comment (bartoszmajsak) [reply get_all_manifests.sh:50]: Colors 🌈

Comment (bartoszmajsak) [reply get_all_manifests.sh:50]: image

**Review by zdtsw — APPROVED** [review]
/lgtm

**Comment by zdtsw** [comment]
lets ensure this will be synced to downstream once odh-dashboard not use odh-manifests

**Pull request merged**